aci/utils.go

```diff
@@ -479,3 +479,40 @@ func toggleOptions(option string) string {
 	}
 	return values[option]
 }
+
+// deleteEmptyKeysFromMap deletes empty keys from a map of string values.
+//
+// If the value associated with the key is either an empty string or "{}". If
+// the value matches either of the conditions, the key-value pair is removed
+// from the map.
+func deleteEmptyKeysFromMap(data map[string]string) {
+	for key := range data {
+		if data[key] == "{}" || data[key] == "" {
```

Comment:
Just for my personal understanding, in what scenario would data[key] be "{}"? Is this where annotation is set to "{}"?